.github/workflows/vhs.yml

```
@@ -1,19 +1,19 @@
 name: 📼 VHS
-
 on:
   push:
     paths:
       - 'docs/tapes/*.tape'
-
   workflow_dispatch:
 
 permissions:
-  contents: write
+  contents: read
```

We need to commit changes

```suggestion
  contents: write
```

---

It's set on the specific job that needs to push commits. See line 15-16.

---

I didn't notice